tests/acceptance/guid-file/registration-file-detail-test.ts

```diff
@@ -26,6 +31,19 @@ module('Acceptance | guid file | registration files', hooks => {
     hooks.beforeEach(function(this: ThisTestContext) {
         this.registration = server.create('registration');
         this.file = server.create('file', { target: this.registration, name: 'Test File' });
+        this.node = server.create(
+            'node',
+            {
+                _anonymized: false,
+            },
```

Comment:
You should only ever need to set anonymized to true since its default state is to be falsy.

Reply:
Ok, this is now removed.